client/app/dbaas/logs/detail/logs-helper.service.js

```diff
@@ -1,12 +1,14 @@
 class LogsHelperService {
-    constructor ($translate, $state, OvhApiDbaas, ServiceHelper, CloudPoll, ControllerModalHelper, LogsConstants) {
+    constructor ($translate, $state, OvhApiDbaas, ServiceHelper, CloudPoll, ControllerModalHelper, LogsConstants, ovhDocUrl) {
         this.$translate = $translate;
         this.$state = $state;
         this.ServiceHelper = ServiceHelper;
         this.CloudPoll = CloudPoll;
         this.ControllerModalHelper = ControllerModalHelper;
         this.LogsConstants = LogsConstants;
         this.OperationApiService = OvhApiDbaas.Logs().Operation().v6();
+        this.ovhDocUrl = ovhDocUrl;
+        this.initGuides();
```

Comment:
Should be called in `$onInit`

Reply:
I think service does not support $onInit lifecycle hook

Reply:
Oops, my bad 😉